README.md

```diff
@@ -18,7 +18,7 @@ Agencies are encouraged to use the following HTML snippet to participate in the
 
 ```html
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
-<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
+<script async id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
```

Comment:
I just removed the `async` attribute from this PR (it's unrelated anyway), but we should look at incorporating @afeld's work and making the DAP snippet properly async-ready.